### ISSUE TITLE
UI element to show Facebook Container is active

### ISSUE THREAD

**Issue by forrie** [mozilla/contain-facebook#836]
It would be useful if the UI could present an element that shows that the Facebook Container is active and being used.  For example, the extension icon could change color?

One problem I see with containers in general is there's no indicator that the tab is or is within a container -- seems like an obvious oversight to me and I've suggested Mozilla address this.   For example, a container icon in the tab, brighter colors, etc.

As for this extension, would it be reasonable to have some UI element show that it's operational and containing the FB site?

**Comment by tukoz** [comment]
Feature request
I won't use Facebook at all unless I know for sure it **is** sandboxed.  

### Actual behavior
Am unable to see whether Facebook Container v2.3.9 is working or if I installed it properly (e.g. still not sure if it requires Container). Opening FB on Firefox with many tabs already opened just opens a new tab with default UI.

Steps to reproduce

- Open Firefox with tabs as usual
- Add a tab and type facebook.com
- Nothing I can see show me whether Facebook Container is working. i.e. the extension icon stays either grey (when kept visible in toolbar) or grey and non visible under the toolbar's "Extensions" button (when moved there to make space).

### Expected behavior
Please show me whether Facebook Container is working as intended or not, e.g. have its icon light up or show color.

### Alternatives considered
Use FB only in a private window, and for the rest pray that `Total Cookie Protection` does what it says.  
This has seemingly worked for me with anything Google related for the last couple years.
 
### My setup 
Using Firefox in a non Private Window.  
Firefox » Preferences/Options » "Tabs": green out but ticked Checkbox "Enable Container Tabs".  
Facebook Container Version: 2.3.9 (also I manually restarted the browser upon installing the extension)  
Operating System + Version: Arch Linux  
Firefox Version: 112.0  
Other installed Add-ons + Version + Enabled/Disabled-Status: Tree Style Tabs and quite a few non-related others (eg no Container, no Conex).
 
### Possibly related issues 
- Firefox Facebook Container Blue Tabs #842 
- Facebook Container color #837

**Comment by siffegh** [comment]
> It would be useful if the UI could present an element that shows that the Facebook Container is active and being used. For example, the extension icon could change color?
> 
> One problem I see with containers in general is there's no indicator that the tab is or is within a container -- seems like an obvious oversight to me and I've suggested Mozilla address this. For example, a container icon in the tab, brighter colors, etc.
> 
> As for this extension, would it be reasonable to have some UI element show that it's operational and containing the FB site?

All container tabs have a wider colored border/strip at the top of the tab and the name/symbol of the container inside the address bar.

**Comment by tukoz** [comment]
You're right @siffegh must have been blinded first time; I now notice the name/symbol (not the border but that's fine).

![thumb][img1]

[img1]: https://nsm09.casimages.com/img/2023/04/29//23042903095018649218167587.png

**Comment by siffegh** [comment]
> You're right @siffegh must have been blinded first time; I now notice the name/symbol (not the border but that's fine).

That's odd; your pic doesn't show a tab at all.

**Comment by forrie** [comment]
From the feedback I've read here and elsewhere, I think folks want more customization of this feature.   User-defined tab colors is an obvious feature that I believe would be easy to implement into the UI.
